core/src/main/java/org/frankframework/http/RestListener.java

```diff
@@ -103,7 +90,7 @@ public void close() {
 	public Message processRequest(Message message, PipeLineSession session) throws ListenerException {
 		HttpServletRequest httpServletRequest = (HttpServletRequest) session.get(PipeLineSession.HTTP_REQUEST_KEY);
 		Message response;
-		String contentType = (String) session.get("contentType");
+		String acceptHeaderThatIsSomeHowStoredUnderThisKey = (String) session.get("contentType");
```

Comment:
Het zou goed zijn als we er ook achter konden komen hoe die header daar komt, en wat het effect ervan is als we die dan niet onder deze naam daar gaan neerzetten maar bijvoorbeeld wat het echt is, `acceptHeader`.

Reply:
Tja, bij de rest listener wordt dit later in de keten het contentType van de response.